examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -4,10 +4,10 @@ import type { Locator } from "playwright";
 
 const _LOCAL_URL = "http://localhost:3001/";
 const CARD_SELECTOR = "div.card";
-const PACKAGE_SELECTOR = "button.package";
-const RC_PAYWALL_TEST_OFFERING_ID = "rc_paywalls_e2e_test";
-const RC_PAYWALL_TEST_OFFERING_ID_WITH_VARIABLES =
-  "rc_paywalls_e2e_test_variables";
+const PACKAGE_SELECTOR = "button.rc-pw-package";
+const RC_PAYWALL_TEST_OFFERING_ID = "rc_paywalls_e2e_test_2";
+const RC_PAYWALL_TEST_OFFERING_ID_WITH_VARIABLES_2 =
```

Comment:
Do we need the `_2` ?
```suggestion
const RC_PAYWALL_TEST_OFFERING_ID_WITH_VARIABLES =
```